rules/windows/image_load/image_load_side_load_antivirus.yml

```diff
@@ -42,5 +42,7 @@ detection:
             - 'C:\Program Files (x86)\McAfee\'
     condition: (selection_bitdefender and not filter_bitdefender) or (selection_fsecure and not filter_fsecure) or (selection_mcafee and not filter_mcafee)
 falsepositives:
-    - Unknown
+    - Applications that load the same dlls mentioned in the detection section. Investigate them and filter them out if a lot FPs are caused.
+    - Dell SARemediation pulgin folder (C:\Program Files\Dell\SARemediation\plugin\log.dll) is known to contain the 'log.dll' file.
```

Comment:
typo: pulgin

Reply:
In general: If you know of specific FPs: Best to filter them using a filter selecection